locations/Bahraich/inputs/simulation/clinic.yaml

```diff
@@ -0,0 +1,59 @@
+---
+################################################################################
+# clinic.yaml - Parameters for specifying a health centre clinic within CLOVER #
+#                                                                              #
+# Author: Ilaria Del Frate                                                     #
+# Copyright: laria Del Frate, 2022                                             #
+# Date created: 27/07/2022                                                     #
+# License: Open source                                                         #
+################################################################################
+
+clinics:
+  # A dash here means one entry, i.e., clinic.
+  - name: kabakedi
+    # Transmission Load
+    floor_area: 56.49 # [m^2]
+    roof_area: 56.49 # [m^2]
+    surface_area_walls: 86.8 # [m^2]
+
+    # Assumption of 2 the open windows covered with glass
+    surface_area_doors_windows: 15.13 # [m^2]
+    u_value_walls: 2.5 # [W/m^2*K] Thermal transmittance of wall
+    u_value_windows_doors: 5 # [W/m^2*K]
+    u_value_floor: 1 # [W/m^2*K]
+    u_value_roof: 0.7 # [W/m^2*K]
+    inside_ideal_temperature: 23 # [ °C]
+    # outside_temperature: 28 #[ °C]
+
+    # Internal heat Load - People
+    people_1: 1.5 #number of nurses present per day
+    people_2: 2.5 #average number of patients and accompaniers
+    heat_loss_people_1: 82 # [W] From Drug Store ASHRAE value
+    heat_loss_people_2: 67.4 # [W] From Apartments ASHRAE value
+    start_time_people_1: [11, 11, 11, 11, 11, 11, 0]
+    end_time_people_1: [18, 18, 18, 18, 18, 18, 0]
+    start_time_people_2: [12, 12, 12, 12, 12, 12, 0]
+    end_time_people_2: [17, 17, 17, 17, 17, 17, 0]
+
+    # Internal heat load - Lighting
+    lamps_internal: 9 # number of lamps within the clinic
+    lamps_external: 4 # number of lamps outside the clinic
+    start_time_lamps: [15, 15, 15, 15, 15, 15, 0] # At 15:00 lights are turned on
+    end_time_lamps: [18, 18, 18, 18, 18, 18, 0]
+    time_lamps_external: 12 # hours per day (night)
+    wattage_lamps: 7 # [W] LED Bulb light A-8901
```

Comment:
It can be a good idea to use CLOVER's functionality for the lamps. The internal ones are averagely on three hours a day during those hours. It depends on a few things and especially on the fact that they are forgotten on most of the times during the working hours, even if the room is not used. The external ones, which are not considered in the cooling load, are on everyday, also Sundays, from 7PM to 7AM. The seasons do not really change the light hours during the year, so that remains pretty stable also for the internal ones.

Reply:
For the LED you are right, I should half it in the calculation. I did some research and I found that still 50% of the energy is producing heat in LEDs (that is very low compared to others) but still there. I can put this info in the literature review or data methodology maybe.